sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -224,9 +224,10 @@ namespace Azure { namespace Core { namespace Http {
     Azure::Core::IO::BodyStream* GetBodyStream() { return this->m_bodyStream; }
 
     /**
-     * @brief A value indicating whether download is happening via stream.
+     * @brief A value indicating whether download will fill a memory buffer or if it will provide a
+     * body stream instead.
```

Comment:
```suggestion
     * @brief A value indicating whether download will return the raw response within a memory buffer or if it will provide a
     * body stream instead.
```